### DISCUSSION TITLE
How to Map Visual Mode Keybindings Like H, J, K, L?

### DISCUSSION THREAD

**Discussion by eeriemyxi** [gdh1995/vimium-c#1094] [Q&A] [answered]
When I do `yv`, I can't select the second line without pressing `j`. Same for the other three main navigation hotkeys. I use Colemak keyboard layout, so it is difficult for me to get used to this behavior. How do I map these keybindings?

This is my config file:
```
map n scrollDown
map e scrollUp

map N scrollPageDown
map E scrollPageUp

mapKey <a-e> <esc>

map l nextTab
map L previousTab

map i scrollRight
map h scrollLeft

map I enterInsertMode

map k performFind
map K performBackwardsFind
```

**Comment by gdh1995** [comment, marked as answer]
Hello, you may use per-mode `mapKey` as said in https://github.com/gdh1995/vimium-c/wiki/Use-in-another-keyboard-layout#per-mode-mapkey .

```
mapKey <s:v> j
mapKey <w:v> k
mapKey <a:v> h
mapKey <d:v> l
# and so on
```

**Reply by eeriemyxi** [reply]
Perfect. Thank you!
